gradle/profile_prod.gradle

```diff
@@ -1,44 +1,34 @@
-apply plugin: 'org.springframework.boot'
-apply plugin: 'com.gorylenko.gradle-git-properties'
-
-ext {
-    logbackLoglevel = "INFO"
+dependencies {
 }
 
-dependencies {
+def profiles = "prod"
+if (project.hasProperty("no-liquibase")) {
+    profiles += ",no-liquibase"
 }
 
-def profiles = 'prod'
-if (project.hasProperty('no-liquibase')) {
-    profiles += ',no-liquibase'
+if (project.hasProperty("swagger")) {
+    profiles += ",swagger"
 }
 
-if (project.hasProperty('swagger')) {
-    profiles += ',swagger'
+springBoot {
+    buildInfo()
 }
 
 bootRun {
     args = []
 }
 
-
 processResources {
-    filesMatching('**/logback-spring.xml') {
```

Comment:
apply the same changes as in profile_dev file

Reply:
done